server/src/routes/api/sequence.mjs

```diff
@@ -0,0 +1,31 @@
+'use strict';
+import { getUserIdFromReq } from '../../util.mjs';
+import {executeSequence} from '../../sequenceManagement.mjs';
+
+//Execute sequence events with respective delay values
+function sendSequence(req, res) {
+    let seqevent = req.body
+    const { ws } = res.locals; // Like magic!
+    const userId = getUserIdFromReq(req);
+
+    //iterating through sequence of events
+    for(let i = 0; i < seqevent.length; i++) {
```

Comment:
I'm still confused by the presence of this loop here. I would expect this code to basically just pass the incoming data structure (in req.body) to the underlying "shared code" in sequenceManagement.mjs.